src/wordpress-plugins/yoast-seo.php

```diff
@@ -320,4 +328,24 @@ protected function reset_free_installation_success_page() {
 	protected function reset_premium_installation_success_page() {
 		WPSEO_Options::set( 'activation_redirect_timestamp', '0' );
 	}
+
+	/**
+	 * Resets the First-time configuration.
+	 *
+	 * @return void
+	 */
+	protected function reset_first_time_configuration() {
+		$workouts_data                  = WPSEO_Options::get( 'workouts_data' );
+		$workouts_data['configuration'] = [ 'finishedSteps' => [] ];
+		WPSEO_Options::set( 'workouts_data', $workouts_data );
+	}
+
+	/**
+	 * Resets the Premium workouts.
+	 *
+	 * @return void
+	 */
+	protected function reset_premium_workouts() {
+		WPSEO_Options::set( 'workouts', [ 'cornerstone' => [ 'finishedSteps' => [] ] ] );
```

Comment:
```suggestion
		WPSEO_Options::set( 'workouts', [ 'cornerstone' => [ 'finishedSteps' => [] ] ] );
		WPSEO_Options::set( 'workouts', [ 'orphaned' => [ 'finishedSteps' => [] ] ] );
```

Reply:
I'd stick to the default value, so it's clear that we are using that one and there is no mismatch